cp3pt0-deployment/setup_tenant.sh

```diff
@@ -382,15 +384,22 @@ function check_singleton_service() {
 function install_nss() {
     title "Checking whether Namespace Scope operator exist..."
 
+    # Extract the numeric part from the CHANNEL variable using parameter expansion
+    local channel_numeric="${CHANNEL#v}"
+    # Check if CHANNEL_NUMERIC is greater than or equal to 4.2, if yes, use NSS channel as v4.2
+    if (( $(echo "$channel_numeric >= 4.2" | bc -l) )); then
+        NSS_CHANNEL="v4.2"
+    fi
```

Comment:
I am wondering if we could avoid hardcode `4.2` and `v4.2` as much as possible?

This is what we have known:
- We already know the latest channel for NSS is `v4.2`, and this is the default value we assign to `NSS_CHANNEL` at the beginning of the script.
- We wanna only update `NSS_CHANNEL` when it has to be updated.
  - The only scenario that we will need to update the `NSS_CHANNEL` is when user using this script to install a old version of CPFS. e.g., `v4.1`.

As a result, we do not need to update `NSS_CHANNEL` when user installs CPFS v4.3+.

```suggestion
    if (( $(echo "$cs_channel < #nss_channel" | bc -l) )); then
        NSS_CHANNEL=$CHANNEL
    fi
```